src/blog/2023/boosting-bpa-development-with-a-high-productivity-platform/boosting-bpa-development-with-a-high-productivity-platform.md

```diff
@@ -0,0 +1,117 @@
+# Boosting BPA development with a high-productivity platform
+
+At [Haulmont](https://haulmont.tech/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article), we’re focused on developing tools and platforms that enhance developers’ productivity by automating coding routines and bringing ready-to-use application components into a single development environment — IntelliJ IDEA. Established in 2008 as a custom development software house, we understand what productivity means for technical leaders. When time and resources are limited, selecting the right tools for the job and capitalising on framework strengths rather than fighting against them is essential. This led to the creation of a high-productivity platform for Java enterprise applications. In 2016 our team introduced the CUBA Platform as open-source, and in 2020, we released the CUBA Platform successor — [Jmix](http://jmix.io/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article).
+
+Currently, [Jmix](http://jmix.io/?utm_source=bpmn&utm_medium=referral&utm_campaign=bpnmio-article) is positioned as an open-source high-productivity platform for building enterprise-grade applications. Java developers worldwide use Jmix to expedite the development of various data-centric, business-process automation, and content management applications, especially when faced with strict time and budget constraints. The most beneficial feature of Jmix is that it enables a single Java developer to become a full-stack developer, delivering a standard Spring Boot application at the end. With no runtime fees, a single language (Java/Kotlin) for the backend and UI, only mainstream core technologies, an IntelliJ IDEA, and a developer-friendly professional IDE, Jmix appeals to developers. Technology interoperability with the IDE ecosystem and the open-source core enabled acquiring a solid community around the technology with over 25 000 developers worldwide.
+
+## First steps with BPMN in the CUBA Platform
+
+Data-centric and content management applications often require business process automation functionality to separate transactional business logic from the code. This can be done with an embedded workflow modeler allowing analysts and developers to design business logic following BPMN standards. We came to this in 2017 when we began receiving requests from our community. Initially, we provided a custom business process design tool powered by Activiti.
```

Comment:
```suggestion
Data-centric and content management applications often require business process automation functionality to separate transactional business logic from the code. This can be done with an embedded workflow modeler allowing analysts and developers to design business logic following BPMN standards. We came to this in 2017 when we began receiving requests from our community. Initially, we provided a custom business process design tool powered by Activiti as part of the CUBA platform, Jmix's predecessor.
```